README.md

```diff
@@ -98,6 +100,11 @@ Additionally, @pika/web runs all dependencies through Babel via `@preset/env` to
   },
 ```
 
+### CLI
+
+| Option    | Description                                                                                    | Example                          |
+|:----------|:---------------------------------------------------------------------------------j-------------|:---------------------------------|
+| `--entry` | Auto-detect `import`s from your codebase. Supports [glob](https://www.npmjs.com/package/glob). | `npx @pika/web --entry src/*.js` |
```

Comment:
I wanted to add this _somewhere_ in the README, but didn’t know where 🤷‍♂ . Open to suggestions

Reply:
Also, tried to nudge the user in the right direction without burdening them with over-explanation. Notice that `src/*.js` glob—it won’t recursively scan your `src` folder and scan too much; just the top-level files in there. Maybe this will achieve the desired results for most?

Reply:
Ah yes, I always find `*` vs. `**` confusing, to me personally and anyone reading any docs I write :) 

Something like `--entry src/**/*.js` without the traversal logic is more explicit about which files are and aren't being analyzed, which as a general rule means less surprises (bugs/issues) for users who might not know what they're doing. It also would really simplify your code, basically letting you skip all of the `buildDependencyTree` complexity. There's a chance it's faster as well, although I haven't tested that. 

The import traversal is definitely really clever, and I really like it in theory. But at least for this first version I'd like to go with explicit over clever for the reasons listed above, if that's alright with you.

Reply:
For the README documentation specifically, I'm happy to merge whatever feels right to you, and then I'll take a cleanup pass PR at the entire README once we get this merged since I think this probably has a larger impact on how to help users get started

Reply:
> The import traversal is definitely really clever, and I really like it in theory. But at least for this first version I'd like to go with explicit over clever for the reasons listed above, if that's alright with you.

Yeah I think that’s a natural progression once adding glob. I see where you’re coming from overall.

> There's a chance it's faster as well, although I haven't tested that.

In my very limited testing, it seemed like the traversal, even with all the razzmatazz of import sniffing, was all really fast. The thing to optimize for from limited testing is **keep the webDependencies to install as minimal as possible.** Installing dependencies that aren’t needed will be the noticeable slowdown for users.

To your point, it should be up to users to notice this, and omit files with deps they don’t need.

What are your thoughts about an `--exclude` or `--ignore` option that overrides `--entry` (and also, in that light, should it be `--entry` or `--include`)?

Reply:
> I'm happy to merge whatever feels right to you

Cool. I’ll just leave alone in this PR, then, but feel free to do a cleanup pass and move/rewrite that info whenever you see fit. I don’t care too much. I think READMEs written in a consistent voice and organization are best.